### PR TITLE
Config changes

### PR DESCRIPTION
**Pull request by kluenderR**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,6 +9,7 @@
   },
   "dependencies": {
     "react": "^18.2.0",
+    "react-devtools": "^4.27.4",
```
Comment (MirkoMajchrzak) [inline package.json:12]: wrong version please update

**Review by JessieHainke — COMMENTED** [review]
nice! :-)

File: src/App.jsx [hunk 2]
```diff
@@ -1,15 +1,16 @@
...
-import logo from "./logo.svg";
-import "./App.css";
+//import "./App.css";
+import Start from "./components/Start";
 
 function App() {
   const [count, setCount] = useState(0);
 
   return (
-    <div className="App">
```
Comment (MirkoMajchrzak) [inline src/App.jsx:9]: but but but please call it App

**Review by MirkoMajchrzak — COMMENTED** [review]
comments added

File: src/App.jsx (hunk 2 above)
Comment (JessieHainke) [reply src/App.jsx:9]: :-)

**Pull request merged**